css-typed-om/Overview.bs

```diff
@@ -3574,6 +3574,11 @@ which will be filled in with more detail as we go.
 
 : <a>align-content</a>
 ::
+    1. If the specified or computed value is the ''normal'' keyword,
```

Comment:
I can either do this, or have separate tables for specified vs computed (will have duplications)?

Reply:
Depends on the property. align-content has the same behavior in specified and computed, so we can combine them.